src/app/containers/Image/index.jsx

```diff
@@ -2,16 +2,26 @@ import React from 'react';
 import { filterForBlockType } from '../../helpers/blockHandlers';
 import { imageModelPropTypes } from '../../models/propTypes/image';
 import Figure from '../Figure';
+import Blocks from '../Blocks';
+import fragment from '../Fragment';
+import InlineLink from '../InlineLink';
 
 const DEFAULT_IMAGE_RES = 640;
 
 const getText = ({ model }) => model.blocks[0].model.blocks[0].model.text;
 
+const componentsToRender = { fragment, urlLink: InlineLink };
 const getCaption = block => {
   if (!block) {
     return null;
   }
-  return getText(block);
+
+  return (
+    <Blocks
+      blocks={block.model.blocks[0].model.blocks[0].model.blocks}
```

Comment:
As discussed in Slack - there _must_ be a way to refactor this 🤔but that's for a future PR.